api/src/main/resources/liquibase.xml

```diff
@@ -94,4 +94,40 @@
     <changeSet author="slubwama" id="patientqueue_1566208965241-17">
         <addForeignKeyConstraint baseColumnNames="creator" baseTableName="patient_queue" constraintName="patient_queue_users_user_id_fk" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="user_id" referencedTableName="users"/>
     </changeSet>
+
+    <changeSet  author="slubwama" id="patientqueue_1566208965241-18">
+        <preConditions onFail="CONTINUE">
+            <tableExists tableName="patient_queue"/>
+            <not><columnExists tableName="patient_queue" columnName="queue_room"/></not>
+        </preConditions>
+        <comment>
+            Add Column queue_room to patient_queue table
+        </comment>
+        <addColumn tableName="patient_queue">
+            <column name="queue_room" type="INT" />
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="slubwama" id="patientqueue_1566208965241-19">
+        <preConditions onFail="CONTINUE">
+            <columnExists tableName="patient_queue" columnName="queue_room"/>
+            <columnExists tableName="queue_room" columnName="queue_room_id"/>
```

Comment:
Do you need to also check for the existence of `queue_room_id`?

Reply:
@dkayiwa Done